java-persistence/src/main/java/com/whitecloak/training/inventory/controller/response/ItemResource.java

```diff
@@ -1,10 +1,20 @@
 package com.whitecloak.training.inventory.controller.response;
 
+import com.whitecloak.training.inventory.model.Category;
+import com.whitecloak.training.inventory.model.Item;
+import com.whitecloak.training.inventory.model.User;
+import com.whitecloak.training.inventory.persistence.entity.CategoryEntity;
+import com.whitecloak.training.inventory.persistence.entity.UserEntity;
+
+import java.util.List;
+
 public class ItemResource {
 
     private long id;
     private String name;
-    private long ownerId;
+    private UserEntity ownerId;
```

Comment:
Entities are leaking to the web layer.

The proper way to do it is to compose this object with `UserResource` and `CategoryResource`.